osu.Game.Tournament/Screens/Board/Components/ModMapSection.cs

```diff
@@ -0,0 +1,177 @@
+                        IconSize = 30,
+                        Text = ModName,
+                    },
+                },
+                mapFlow = new FillFlowContainer<FumoBeatmapPanel>
+                {
+                    Name = @"Map pool content",
+                    Anchor = Anchor.TopCentre,
+                    Origin = Anchor.TopCentre,
+                    RelativeSizeAxes = Axes.X,
+                    AutoSizeAxes = Axes.Y,
+                    Spacing = new Vector2(10),
+                    Padding = new MarginPadding { Horizontal = 10 },
+                },
+                placeholderFlow = new FillFlowContainer
+                {
+                    Name = @"Placeholder",
+                    Anchor = Anchor.TopCentre,
+                    Origin = Anchor.TopCentre,
+                    RelativeSizeAxes = Axes.X,
+                    AutoSizeAxes = Axes.Y,
+                    Spacing = new Vector2(10),
+                    Padding = new MarginPadding { Horizontal = 10 },
+                    Children = new Drawable[]
+                    {
+                        new SpriteIcon
+                        {
+                            Anchor = Anchor.Centre,
+                            Origin = Anchor.Centre,
+                            Icon = FontAwesome.Solid.ExclamationCircle,
+                            Size = new Vector2(24),
+                        },
+                        new TournamentSpriteText
+                        {
+                            Anchor = Anchor.Centre,
+                            Origin = Anchor.Centre,
+                            Text = BaseStrings.NoBeatmapAvailable,
+                            Font = OsuFont.Torus.With(weight: FontWeight.SemiBold, size: 24),
+                        },
+                    },
+                },
+            };
+        }
+
+        protected override void LoadComplete()
+        {
+            base.LoadComplete();
+
+            ladder.CurrentMatch.BindValueChanged(matchChanged);
+            ladder.CurrentMatch.Value?.Round.BindValueChanged(_ => updateList());
+            ladder.CurrentMatch.Value?.Round.Value?.Beatmaps.BindCollectionChanged((_, _) => updateList());
+
+            updateList();
+        }
+
+        private void matchChanged(ValueChangedEvent<TournamentMatch?> e)
+        {
+            if (e.OldValue != null)
+            {
+                e.OldValue.ChessPlacements.CollectionChanged -= placementChanged;
+            }
+
+            if (e.NewValue != null)
+            {
+                e.NewValue.ChessPlacements.CollectionChanged += placementChanged;
+                e.NewValue.Round.BindValueChanged(_ => updateList());
+                e.NewValue.Round.Value?.Beatmaps.BindCollectionChanged((_, _) => updateList());
+            }
+
+            updateList();
+        }
+
+        private void placementChanged(object? _, NotifyCollectionChangedEventArgs __)
+            => Scheduler.AddOnce(updateList);
```

Comment:
don't update list when placement changed
it will re-create FumoBeatmapPanel and cancel the animation